<launch>

  <node pkg="laser_slam_mapper" type="mapper" name="topological_mapper"
        output="screen">
    <param name="reference_frame" value="graph_optimization"/>
    <param name="grid_size" value="15.0"/>
    <param name="grid_overlap" value="0.5"/>

    <!-- Used for clearing out known free areas -->
    <param name="robot_radius" value="0.3"/>
  </node>


</launch>